ui/src/app/app.component.html

```diff
@@ -36,6 +36,7 @@
 <div class="container dataflow-main-container">
   <div class='row'>
     <div class='col-sm-24'><ng2-toasty></ng2-toasty>
+      <ng-template [ngBusy]="{busy: busy, minDuration: 500}"></ng-template>
```

Comment:
Just curious.  We are waiting consistently 500 ms whether we need to or not.  Is there a way we don't have to wait?

Reply:
Also Spinner does not stay present when waiting for data to return from server.   It leaves after 500ms.

Reply:
@cppwfs Before, we waited 2secs which was way too long. The spinner in its current form should have a min duration - otherwise you end up with a bad flicker in certain situations (bad UX). I hope 500ms is a decent compromise (maybe 400ms?). Down the road we may want to switch to a better more efficient "loading indicator". E.g. in the case of a thin horizontal progress-bar (placed under the top-navbar, we would not need that min-duration IMO)  CC @oodamien

Reply:
Understood.  Thanks